android/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="apptentive_distribution">React Native</string>
```

Comment:
This should be 5.8.3 for android. I missed it in the previous PR.

Reply:
(I forgot to restore this file in the previous PR). This is sent as part of the app release object and should reflect the React Native SDK version, not the native SDK versions.